scripts/c2r_script.py

```diff
@@ -558,7 +610,10 @@ def transform_raw_data(raw_data):
 
 
 def update_insights_inventory():
-    """Call insights-client to update insights inventory."""
+    """
+    Call insights-client to update insights inventory.
+    SCRIPT_TYPE == "CONVERSION" specific.
```

Comment:
Don't think we need this one here... We already have a comment and a condition in main before calling the function. 
```suggestion
```